projects/README.md

```diff
@@ -0,0 +1,9 @@
+# Instructions for per-platform builds
```

Comment:
According to [pitchfork layout](https://api.csswg.org/bikeshed/?force=1&url=https://raw.githubusercontent.com/vector-of-bool/pitchfork/develop/data/spec.bs) (mentioned in README.md) this whole folder should be `<root>/tools/build` rather than `<root>/projects`.

Reply:
I will update the layout to follow pitchfork guidelines.

Reply:
The existing `<root>/tools/build` folder contains a script: `build.ps1`. With the addition of my subfolders, I think this file should be moved to its own subfolder. What do you think? Additionally, the project root contains `Microsoft.MixedReality.WebRTC.sln`, seems like it should exist under `tools` as well?

Reply:
Yes agreed. Although I am still unclear on what approach to take for multi-platform build scripts. I would rather rely on a maintained solution like `CMake` or `gn` rather than roll our own scripts for all platforms.

The Visual Studio solution was at the root for discoverability, because (and these points should ideally be fixed):
- we do not ship binaries/packages for everything (Unity integration ships as source only for now, we should work on that to make it more user friendly)
- there was a single platform
- I don't think it is very intuitive to go look into the `tools/build` folder to find a Visual Studio solution for the project

Not sure how we can maintain good discoverability while moving everything into `tools/build`. Maybe have a single `build.sh`/`build.ps1` script at root if we go that way. Or maybe have a single `CMakeLists.txt` at root if we use CMake, and then have everything else inside `tools/build`. I'm open to suggestions.

Reply:
`gn` might be an appropriate build system for this repo, given the growing number of platforms and options you plan to support. I am new to `gn` and `Ninja`, but the system seems very flexible with regard to build configuration and conditional build of components. I don't know how easily it can be coerced into building Visual Studio projects.

Reply:
It does support generating some VS solution via the `--ide` flag. But no idea how "good" that is (can we regenerate the exact same solution we hand-crafted so far?). My other concern is that I have yet to understand how to integrate an existing `gn` project (libwebrtc) into another one (mr-webrtc), as ideally, we'd take advantage of that to have a single unified solution for building.